### PR TITLE
Added subscription calls start/stop and get plans

### PR DESCRIPTION
**Pull request by damirius**
Two new calls for rating subscriptions

### CODE REVIEW

File: docs/calls/rating_subscription_plans_get.rst [hunk 1]
```diff
@@ -0,0 +1,198 @@
+.. highlight:: js
+
+.. _calls-ratingsubscriptionplansget-docs:
+
+Get Rating Subscription Plans
```
Comment (schemar) [inline docs/calls/rating_subscription_plans_get.rst]: What are "rating" subscription plans ⁉️

File: docs/calls/user_rating_subscription.rst [hunk 2]
```diff
@@ -0,0 +1,106 @@
+.. highlight:: js
+
+.. _calls-userratingsubscription-docs:
+
+User Rating Subscription
```
Comment (schemar) [inline docs/calls/user_rating_subscription.rst]: this name is not very readable and is not in line with the existing OIOI calls. Please change it.

File: docs/calls/user_rating_subscription.rst [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+        A token can be used to authenticate the user (string).
+
+        For example: if the identifier type is username and the identifier is the user's username,
+        then token is used for authentication instead of a password.
+
+id
+    ID of Subscription Plan for which we want to start/cancel subscription (int).
+
+subscribe
+    `true` if we want to subscribe. `false` if we want to cancel subscription (bool).
```
Comment (schemar) [inline docs/calls/user_rating_subscription.rst]: when you think about it from a natural language point of view, "subscribe: false" does not mean that a subscription shall be cancelled.

File: docs/calls/rating_subscription_plans_get.rst (hunk 1 above)
Comment (damirius) [reply docs/calls/rating_subscription_plans_get.rst]: There are already subscription calls in OIOI for old subscriptions.

File: docs/calls/user_rating_subscription.rst (hunk 2 above)
Comment (damirius) [reply docs/calls/user_rating_subscription.rst]: Any ideas?

File: docs/calls/user_rating_subscription.rst (hunk 3 above)
Comment (damirius) [reply docs/calls/user_rating_subscription.rst]: Took it from the subscription google doc.

**Comment by schemar** [comment]
In my opinion, this version of OIOI should only have the new subscriptions and therefore can replace the old subscriptions. @v03adk what do you think?

**Comment by v03adk** [comment]
@schemar @damirius Definitely we can replace old subscriptions here. 
As for naming, I would keep `RatingSubscription` to keep consistency with Rating Engine

**Comment by damirius** [comment]
@schemar  @v03adk Is `UserManageRatingSubscription` OK for the call then? Call wil be `user-manage-rating-subscription`, also `subscribe` field can be renamed to `cancel`. It will still be boolean, if it's set to `true` we will cancel subscription like we do now, if it isn't set or if it's false we will try to create new subscription for the plan instead?

**Comment by schemar** [comment]
Why does there have to be "rating" in the name? A consumer of the API knows nothing about "rating". This is just confusing!

**Comment by v03adk** [comment]
Indeed. Agree with you

**Comment by damirius** [comment]
OK, so everything the same except it's UserManageSubscription? Change other calls to remove Rating from them too

**Comment by schemar** [comment]
sounds good

**Comment by damirius** [comment]
Do we need `user` prefix for other two calls? Plan listing would be then `user-subscription-plans-get` and get active would be `user-subscription-get-active`. Or is it enough just to remove `rating` like we mentioned, so they would look like `subscription-plans-get` and `subscription-get-active`?

**Comment by schemar** [comment]
I think both is fine. @v03adk should decide as owner of the OIOI.

**Comment by v03adk** [comment]
I would vote for `user-subscription-plans-get` and `user-subscription-get-active`

**Comment by damirius** [comment]
I think they are called a bit different, the bigger problem is that they will have a different requests/responses which will still break stuff on consumer apps.

**Comment by v03adk** [comment]
sorry, looks like I got lost in this conversation :(  
so, we have two calls and want completely replace old calls with them? or leave old ones and introduce 2 new calls?

**Comment by schemar** [comment]
doesn't matter

**Pull request merged**